src/sections/Meshery/Meshery-integrations/Integration.style.js

```diff
@@ -191,3 +192,46 @@ export const IntegrationCard = styled.div`
     }
   }
 `;
+
+export const IntegrationSlider = styled(Slider)`
+  .slick-prev:hover, .slick-next:hover{
+    box-shadow: none;
+    outline:none;
+  }
+
+  .slick-arrow:hover:before {
+    color:#00b39f;
+  }
+
+  .slick-arrow {
+    margin:1rem 0 0 -.7rem;
+  }
+
+  .slick-slide {
+    width: auto !important;
+    margin: 0 .5rem;
+  }
+
+  .slick-prev::before {
+    content:"\u2039";
```

Comment:
Okay i don't have safari web browser to check this out but changing this unicode of slick-prev and slick-next to this character might work ‹ ›

Reply:
Please use the same icon that we use for the sliders at other places in the site.
Do not add a new icon

Reply:
yeah it is the same one as used in special-section this two ‹ ›